packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -387,6 +387,253 @@ void runAllTests({ required bool useMaterial3 }) {
       expect(tester.getTopLeft(find.text('label')).dy, tester.getTopLeft(find.text('hint')).dy);
       expect(tester.getBottomLeft(find.text('label')).dy, tester.getBottomLeft(find.text('hint')).dy);
     }
+
+    //InputDecorator input/label text layout with floatingLabelGap: appears above input text
```

Comment:
Nit: Space after the `//`, here and below.

Reply:
done